### ISSUE TITLE
Parser crashes when adding a { in RAML 0.8

### ISSUE THREAD

**Issue by agustinlg** [raml-org/raml-js-parser-2#586]
Only happens with RAML 0.8 when adding the `{` before the `:`

```
#%RAML 0.8
title: Bug with { in RAML 0.8
/newResource{:
  displayName: resourceName
```

**Comment by ddenisenko** [comment]
Fixed. Fix should be published in 1.1.12

**Comment by agustinlg** [comment]
Verified
